modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -552,8 +608,11 @@ public void testGetIdP() throws IOException {
                 .body("description", equalTo("IDP for Google Federation"))
                 .body("isEnabled", equalTo(true))
                 .body("isPrimary", equalTo(false))
-                .body("federatedAuthenticators.authenticators.find { it.authenticatorId == '" +
-                        SAMPLE_FEDERATED_AUTHENTICATOR_ID + "' }.definedBy", equalTo("SYSTEM"))
+                .body(baseIdentifier + "name", equalTo(new String(Base64.getDecoder().
+                        decode(SAMPLE_FEDERATED_AUTHENTICATOR_ID))))
+                .body(baseIdentifier + "isEnabled", equalTo(true))
+                .body(baseIdentifier + "definedBy", equalTo(DEFINED_BY_SYSTEM))
+                .body(baseIdentifier + "tags", Matchers.hasItems("Social-Login", "APIAuth"))
```

Comment:
check self as well

Reply:
This is updated with 7831a169607232cefc26307b20c8ca3746542a52